Tools/SwiftBrowser/Source/ContentView.swift

```diff
@@ -55,6 +55,67 @@ fileprivate struct ToolbarBackForwardMenuView: View {
     }
 }
 
+fileprivate struct DialogActionsView: View {
+    private let dialog: DialogPresenter.Dialog
+
+    @State private var promptText = ""
+
+    init(dialog: DialogPresenter.Dialog) {
+        self.dialog = dialog
+
+        if case let .prompt(_, defaultText, _) = dialog.configuration, let defaultText {
+            _promptText = State(initialValue: defaultText)
+        }
+    }
+
+    var body: some View {
+        switch dialog.configuration {
+        case let .alert(_, dismissAlert):
+            Button("Dismiss") {
+                dismissAlert()
+            }
```

Comment:
nit: The current button for JS alerts is "Close," parity might be good here? Also not sure if it matters

Reply:
ya it doesn't really matter since this is just a test app, but I can change it